linux/installer/common/gen_source/gen_source.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Please discard the python changes in this PR. We can change it later or in another PR.